src/features/editProfile/model/editProfileFormSchema.ts

```diff
@@ -0,0 +1,29 @@
+import { z } from "zod";
+
+export const editProfileFormSchema = z.object({
+  username: z
+    .string()
+    .min(1, { message: "Обязательное поле" })
+    .max(30, { message: "Максимальная длина - 30 символов" })
+    .nullable(),
+  image_url: z.string().optional(),
+  city: z
+    .string()
+    .min(1, { message: "Обязательное поле" })
+    .max(30, { message: "Максимальная длина - 30 символов" })
+    .nullable(),
+  gender: z.string().optional(),
+  is_private: z
+  .boolean({ required_error: 'Обязательное поле', invalid_type_error: 'Обязательное поле' }),
+  bio: z
+    .string()
+    .min(1, { message: "Обязательное поле" })
+    .max(410, { message: "Максимальная длина - 410 символов" })
+    .nullable(),
```

Comment:
Server does not accept bio equal to null, nullable not need here. When you substitute fields from server you needs to set form state like this: bio ?? ""